NitroxClient/GameLogic/Logic.cs

```diff
@@ -17,6 +17,7 @@ public class Logic
         public Cyclops Cyclops { get; private set; }
         public Interior Interior { get; private set; }
         public MobileVehicleBay MobileVehicleBay { get; private set; }
+        public ChunkSerializer Serializer { get; private set; }
```

Comment:
Can we rename this Chunks?  Multiplayer.Logic.Chunks is more explicit as to what this is doing.